charts/karmada/values.yaml

```diff
@@ -56,15 +56,6 @@ kubectl:
     ##
     pullPolicy: IfNotPresent
 
-## pre-install job config
```

Comment:
There three lines are due to https://github.com/karmada-io/karmada/pull/2348#discussion_r943093439.

Reply:
ok

Reply:
done